piptools/resolver.py

```diff
@@ -193,11 +201,6 @@ def __init__(
         allow_unsafe: bool = False,
         unsafe_packages: set[str] | None = None,
     ) -> None:
-        """
-        This class resolves a given set of constraints (a collection of
```

Comment:
A docstring here may still be useful. It could describe what this initializer method does.